src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ConstructorSymbolReferenceFinder.cs

```diff
@@ -190,15 +190,26 @@ private static bool IsPotentialReference(PredefinedType predefinedType, ISyntaxF
             FindReferencesInDocumentUsingIdentifier(symbol, simpleName, state, processResult, processResultData, cancellationToken);
     }
 
-    private void FindReferencesInImplicitObjectCreationExpression<TData>(
+    private static void FindReferencesInImplicitObjectCreationExpression<TData>(
         IMethodSymbol symbol,
         FindReferencesDocumentState state,
         Action<FinderLocation, TData> processResult,
         TData processResultData,
         CancellationToken cancellationToken)
     {
+        if (!state.Cache.SyntaxTreeIndex.ContainsImplicitObjectCreation)
+            return;
+
+        // Note: only C# supports implicit object creation.  So we don't need to do anything special around case
```

Comment:
>So we don't need to do anything special around case [](http://example.com/codeflow?start=61&length=51)

dumb question. If we don't need to worry about VB here, and this is the only caller of GetNewKeywordTokens, why does that code need to worry about VB?

Reply:
it technically doesn't.  it was just easier this way.